CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to UCL API
```

Comment:
Something like the PR one we already have or the one an existing OSS project uses? Won't have time to make one until like next week :/